src/borg/archiver.py

```diff
@@ -2665,42 +2687,40 @@ def do_break_lock(self, args, repository):
             # note: using = is essential to avoid cmdline argument parsing issues.
             borg create --pattern=+pics/2018/good --pattern=-pics/2018 repo::arch pics
 
+            # backup only JPG/JPEG files (case insensitive) in all home directories:
+            borg create --pattern 're:\\.jpe?g(?i)' repo::arch /home
+
```

Comment:
maybe a more realistic example would be to exclude all *.ISO (they usually are downloads **and** big).

guess most people do not **only** want to backup their jpegs, but also other files in their homes.

Reply:
I just realize that there is a missing '+' in 're:'...
I wanted to include a simple example for the use case of #5310 question (back up files of distinct extension).
Of course, we can add more common useful examples. I'll add something...